### PR TITLE
#27 Create Login UI

### PR DESCRIPTION
**Pull request by Marktine**
- Add `styled-component` for styling components.
- Add `react-hook-form` for form values validation
- Initiate Login form UI.
- Add validators to validate `username` and `password` as mentioned in issue #27

### CODE REVIEW

File: dmd-frontend/src/pages/login/index.tsx [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+            validate: (value: string) => {
+                if (value.length === 0) return 'Password cannot be empty';
+                if (value.length < 8 || value.length > 32) return 'Password must be between 8 and 32 characters';
+                return checkPassword(value) || 'Invalid password, please have a look at our password policy';
+            }
+        });
+    }, [register]);
+
+    return (
+        <FlexContainer
```
Comment (chiro-hiro) [inline dmd-frontend/src/pages/login/index.tsx:59]: Who the hell using this code style?

Comment (Marktine) [reply dmd-frontend/src/pages/login/index.tsx:59]: @chiro-hiro , Which line exactly ?

Comment (Marktine) [reply dmd-frontend/src/pages/login/index.tsx:59]: If it's about those that are in `useEffect` then I'm planning to move them out.

**Comment by chiro-hiro** [comment]
```
                        <Grid
                            container
                            direction="column"
                        >
```
Don't use this code style please, use this:
```
                        <Grid container direction="column">
```

File: dmd-frontend/src/pages/login/index.tsx (hunk 1 above)
Comment (chiro-hiro) [reply dmd-frontend/src/pages/login/index.tsx]: @Marktine Line 58 and the rest of file :|.

Comment (Marktine) [reply dmd-frontend/src/pages/login/index.tsx]: > ```
>                         <Grid
>                             container
>                             direction="column"
>                         >
> ```
> 
> Don't use this code style please, use this:
> 
> ```
>                         <Grid container direction="column">
> ```
Ohhh, Okay, I'll modify them

**Pull request closed without merging**